### DISCUSSION TITLE
Homey integration available now

### DISCUSSION THREAD

**Discussion by gruijter** [Koenkk/zigbee2mqtt#18142] [Show and tell]
Although very much in development, I would like to inform everyone there is a Homey app now available. Information and a link to the Homey app can be found here: https://community.homey.app/t/83214

@Koenkk Maybe you can add the Homey integration to your readme?
I'm still struggling to find the best way to map all Zigbee2MQTT 'exposures' to Homey 'capabilities'. Similar work has been done already for HA. Is there an approach you would advise me to use?
E.g. Is there a complete list of all possible exposures? Or would this be the place to find that: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/extension/homeassistant.ts

Thx!

**Comment by Koenkk** [comment]
- Feel free to make a pr for the README.md (click edit icon here)
- All the possible exposes are documented here: https://www.zigbee2mqtt.io/guide/usage/exposes.html#specific
